### ISSUE TITLE
Screen reader only works with keyboard nav

### ISSUE THREAD

**Issue by etwa4650** [phetsims/gravity-force-lab-basics#143]
For https://github.com/phetsims/QA/issues/333

<b>Test Device</b>

  JORDAN

  <b>Operating System</b>

Mac OS 10.14.5 

  <b>Browser</b>

  Safari 12.1.1

  <b>Problem Description</b>

When moving the weights around using a mouse, VoiceOver does not read anything.

Works fine when editing mass weights and checking/unchecking boxes.

Also occurs with Windows 10: NVDA & JAWS
  

  <b>Steps to Reproduce</b>

  1: Turn on Screen reader 
  2: Go to Gravity Force Lab Basics
  3: Drag the weights around

**Comment by zepumph** [comment]
@jessegreenberg and I discussed this, and we are already aware of this. We don't support this currently. The reasoning is that a lot of the screen reader verbage is coming from the PDOM, not from the sim model. We have not worked on this mouse/screen reader use case at all to this point. Tagging @emily-phet to make sure that she agrees. Marking as won't fix. @etwa4650 please update the appropriate spots in QA literature, you don't need to test with traditional input and a screen reader unless specifically mentioned in the test task.

**Comment by zepumph** [comment]
assigning to @etwa4650, feel free to close when understood and documented. Let me know if you have any questions.

**Comment by terracoda** [comment]
@etwa4650, if you want to use the trackpad to interact with a slider (move a sphere or change mass) on MacOS, there is a special interaction command when u sing VoiceOver. VoiceOverr will announce changes while using the "interaction command" and the track pad.

**To start interacting:**
- Press VO-Shift-Down Arrow. If you’re using VoiceOver gestures, flick right with two fingers on the trackpad.

**To stop interacting:**
- Press VO-Shift-Up Arrow. If you’re using VoiceOver gestures, flick left with two fingers on the trackpad.

**Comment by terracoda** [comment]
VO = control + option keys

**Comment by terracoda** [comment]
Also, note that when "interacting" with or using sliders it might be easy to push yourself into "Quick Nav" mode. Make sure Quick Nav is off. With Quick Nav on, your arrow keys only navigate/read the content, they don't change values.

**Comment by etwa4650** [comment]
@terracoda @zepumph Thank you. Ill go ahead and close this issue.
